sdks/python/run_pylint.sh

```diff
@@ -64,6 +64,7 @@ echo "Running pycodestyle for module $MODULE:"
 pycodestyle "$MODULE" --exclude="$FILES_TO_IGNORE"
 echo "Running flake8 for module $MODULE:"
 flake8 $MODULE --count --select=E999 --show-source --statistics
```

Comment:
My sense is that __E901,E999,F821,F822,F823__ are the "showstopper" flake8 issues that can halt the runtime with a SyntaxError, NameError, etc.  The other flake8 issues are merely "style violations" -- useful for readability but they do not really effect runtime safety.

I would therefore suggest replacing __--select=E999__ with __--select=E901,E999,F822,F823__ because the codebase currently passes all of these tests and we would like to avoid any backsliding.  We would also want to add __F821__ (undefined names) to that list but work must be completed on __basestring, buffer(), cmp(), file(), unicode, xrange()__, and #4561 before that can be done.

@holdenk

Reply:
Hi, did you mean to make a comment on this PR?

Reply:
Yes.  As long as changes are being made, this is a good one to make as well.  Just want to avoid backsliding.